### DISCUSSION TITLE
Sugerencias Dra. Rosario

### DISCUSSION THREAD

**Discussion by ma-robles** [grupoioa/gis_agm_series#81] [Ideas]
Aquí les pongo las sugerencias que me envió la Dra. Rosario

1. Poner ‘Promedio Diario’ en lugar de ‘Diaria’
2. No se podrían dejar fijos los colores para cada serie de tiempo, es algo confuso que cambien cada vez que se agrega una variable
3. Agregar las coordenadas de los puntos y el nombre de la variable en el archivo CSV
4. Faltaría poner las opciones de Promedios mensuales para las temperaturas máximas y las Mínimas absolutas diarias
5. En las gráficas, se podrían agregar unas líneas verticales para separar los meses y que las iniciales del mes quedaran centradas?
6. Checar los valores de los Promedios de máximas absolutas mensuales, ya que la línea no se ve centrada con respecto a los valores diarios.
7. No se despliega la Máxima absoluta por mes en Viento.
8. Creo que en lugar de 'punto_X' se podría poner solamente 'P_X' para no ocupar tanto espacio.
9. Revisar también los Promedios de Máx. Abs. Mensuales del Viento.

**Comment by AbrilIchel** [comment]
Listo! ya las ubique las integro al documento
